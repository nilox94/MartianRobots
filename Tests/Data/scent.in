2 2
0 0 E
FFF
0 0 E
FFF
2 0 N
FFF
2 0 N
FFF
2 2 W
FFF
2 2 W
FFF
0 2 S
FFF
0 2 S
FFF
