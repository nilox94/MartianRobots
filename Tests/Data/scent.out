2 0 E LOST
2 0 E
2 2 N LOST
2 2 N
0 2 W LOST
0 2 W
0 0 S LOST
0 0 S
